css/properties/cursor.json

```
@@ -32,7 +32,10 @@
             "safari": {
               "version_added": "1.2"
             },
-            "safari_ios": "mirror",
+            "safari_ios": {
+              "version_added": "13.4",
+              "notes": "This property is only supported on iPads with a trackpad connected. Only a small subset of values are supported; all other values will use the <code>default</code> pointer as a fallback."
```

To confirm the completeness of this note: a Bluetooth mouse doesn't work, only trackpads?

Also, it can get messy when feature data summarizes its descendants (see https://github.com/mdn/browser-compat-data/pull/24502 for a more serious version of this). I'm suggesting dropping that, since it's more likely to go stale if the details ever change.

```suggestion
              "notes": "This property is only supported on iPads with a trackpad connected."
```

---

Ah, right, a Bluetooth mouse would also present the same results!

I'm also thinking that the summary of behavior is alright for this property, because it's not describing behavior that applies to a specific property or element, but a general behavior that applies to all the values besides a few.

---

In that case, about how about this version instead that avoids being too specific about the scale of the issue?

Also I realize now that the two notes are really about distinct issues:

```suggestion
              "notes": [
                "This property is only supported on iPads with an external pointing device.",
                "Unsupported values use the <code>default</code> pointer as a fallback."
              ]
```